compiler/modules/types/table.bal

```diff
@@ -0,0 +1,61 @@
+public function tableDef(SemType constituentType) returns SemType {
```

Comment:
We need a better name for this function.  Let's call it `tableContaining` (but if you have a better idea, please propose it).

Use `memberType` not `constituentType` (the latter is only for sequences).